contracts/deploy/mainnet/106_ousd_metamorpho_usdc.js

```diff
@@ -0,0 +1,98 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "106_ousd_metamorpho_usdc",
+    forceDeploy: false,
+    // forceSkip: true,
+    // reduceQueueTime: true,
+    deployerIsProposer: true,
+    proposalId:
+      "47304916506185581761087082198807005006233194835954133169998049547086254898248",
```

Comment:
This seems to have creeped up again 😅

Reply:
Again? I just tried and I have the same id. 
How do you get the Id? 
I get it by running `yarn run node` or `yarn run test:fork` and reading the logs.
<img width="804" alt="Capture d’écran 2024-09-05 à 14 00 19" src="https://github.com/user-attachments/assets/83b3a8ee-d279-4b63-9117-1c5fd8f92032">

Reply:
This would usually go in the deploy PR?

Reply:
And you edit it after? 
Ok no problem 😉